docs/guide/docker-builder-vm.md

```diff
@@ -90,7 +90,12 @@ Cirrus CI will check if a container was already built, and if so, Cirrus CI will
 Under the hood, for every `Dockerfile` that is needed to be built, Cirrus CI will create a Docker Builder task as a dependency. 
 You will see such `build_docker_image_HASH` tasks in the UI.
 
-!!! info "Using with private GKE clusters"
+!!! warning "Danger of using `COPY` and `ADD` instructions"
+    Cirrus doesn't include files added or copied into a container image in the cache key. This means that for a public repository
+    a potential bad actor can create a PR with malicious scripts included into a container, wait for it to be cached and then
+    reset the PR so it looks harmful.
```

Comment:
That doesn't sound quite right... you mean reset the PR so it doesn't look harmful?

Reply:
Reset PR branch. Like modify scripts that are added to not look harmful.

Reply:
I think @RDIL meant "s/harmful/not harmful/". Maybe it's good to add one or two sentences about the consequences, e.g., what a malicious script could do.

Reply:
Oh, yeah, some other folks noticed it and I fixed the harmfull thing in 51474e16c3939a0608b5acb681d6ab21b9ce70d8